.travis.yml

```diff
@@ -0,0 +1,6 @@
+sudo: false
+language: scala
+jdk: oraclejdk8
+script:
+- git clone -b 2.12.x https://github.com/scala/scala
```

Comment:
oh, not a good idea actually ;-)

it needs the history, the commands it issues are `git log` and `git show`